backend/configs/usr/lib/cgi-bin/check.sh

```
@@ -0,0 +1,45 @@
+jq -e '.probeOpenPorts==true' /etc/wb-security.conf  >/dev/null || {
+    mqtt_send_not_found
+    send_output 200 "Check disabled in config"
+    exit 0
+}
+
+
+SERIAL=$(cat "/var/lib/wirenboard/short_sn.conf")
+URL="${HTTP_SCHEME}://${HTTP_HOST}/"
+
+data=$(curl --connect-timeout 120 --max-time 120 --silent --show-error --fail -X POST -o /dev/stdout -d "serial=${SERIAL}&url=${URL}" "${CHECK_SERVER}/probe/")
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)